go/runner.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	inFile := os.Args[1]
+	outFile := os.Args[2]
+	fmt.Printf("Reading from: %v.  Writing to: %v\n", inFile, outFile)
+	dat, err := ioutil.ReadFile(inFile)
+	if err != nil {
+		panic(err)
+	}
+	lines := strings.Split(string(dat), "\n")
+	items = nil
+	for i := 0; i < len(lines); i++ {
+		if !strings.HasPrefix(lines[i], ";") && (lines[i] != "") {
+			pieces := strings.Split(lines[i], "__")
+			name := pieces[0]
+			sellIn, _ := strconv.Atoi(pieces[1])
+			quality, _ := strconv.Atoi(pieces[2])
+			items = append(items, Item{name, sellIn, quality})
+		}
+	}
+
+	GildedRose()
```

Comment:
I think we should fix how items are passed to `GildedRose`. Many people fix this problem with the old code, and this runner would make it more difficult.

It should be :
```go
func GildedRose(items []Item) {
...
}
```